go/vt/vtctl/vtctl.go

```diff
@@ -1896,6 +1896,10 @@ func commandReshard(ctx context.Context, wr *wrangler.Wrangler, subFlags *flag.F
 	cells := subFlags.String("cells", "", "Cell(s) or CellAlias(es) (comma-separated) to replicate from.")
 	tabletTypes := subFlags.String("tablet_types", "", "Source tablet types to replicate from.")
 	skipSchemaCopy := subFlags.Bool("skip_schema_copy", false, "Skip copying of schema to targets")
+
+	startStopped := subFlags.Bool("start_stopped", false, "Do not start streams: streams start off in the Stopped state")
```

Comment:
This name can be misleading. It means "Start streams in stopped state", but it could mean "Start the streams that are in stopped state". Do you think the flag documentation is sufficient to avoid confusion?

Reply:
Yeah, I don't like `start_stopped` much either, but could only think of a verbose `start_workflow_in_a_stopped_state`  ...

The workflow state can only be  one of Stopped or Running when we create the workflow and Running is the default, so we would only have one useful value for `-initial_state`.

Reply:
Agree that the name is confusing. Perhaps: `-start-idle`? or `-initially-stopped`?

Reply:
We could do the inverse, and name the flag `autorun`. Or keep the logic the same and name the flag `-no_autorun` (or s/run/start, whatever you prefer!)

Reply:
I like `auto_run` / `auto_start`.

Reply:
Great idea about the inverse flag. Used `auto_start` (default: true)